src/fugw/mappings/sparse_barycenter.py

```diff
@@ -181,6 +183,9 @@ def fit(
             have the same number of features n_features.
         geometry_embedding (np.array or torch.Tensor): Common geometry
             embedding of all individuals and barycenter.
+        subject_weights (list of float, optional): Weights of each individual.
```

Comment:
Naive question: is there really a usecase for this parameter ?

Reply:
Mostly for interpolation:
https://pythonot.github.io/auto_examples/gromov/plot_gromov_barycenter.html

In our fMRI application, all subjects are assigned the same weight. However other FUGW users might do fancier things...

Reply:
I removed it since there is no immediate usecase where it could be relevant for the end user.